systemverilog-plugin/UhdmAst.cc

```diff
@@ -3713,17 +3713,31 @@ void UhdmAst::process_tagged_pattern()
     AST::AstNode *lhs_node = nullptr;
     if (assign_node) {
         assign_type = assign_node->type;
-        lhs_node = assign_node->children[0];
+        lhs_node = assign_node->children[0]->clone();
     } else {
         lhs_node = new AST::AstNode(AST::AST_IDENTIFIER);
         auto ancestor = find_ancestor({AST::AST_WIRE, AST::AST_MEMORY, AST::AST_PARAMETER, AST::AST_LOCALPARAM});
         if (!ancestor) {
+            // special case: if this assignment pattern is inside cell high conn port
+            if (find_ancestor({AST::AST_CELL})) {
```

Comment:
Is it possible to add `AST::AST_CELL` to `find_ancestor` 3 lines above and reorganize the condition into something like:
```
if (!ancestor) {
    log_error(...);
}
if (ancestor->type == AST::AST_CELL) {
    ...
}
```
Reason: less nesting and one `find_ancestor` less.